ksql-common/src/main/java/io/confluent/ksql/function/IndexableAsFunction.java

```diff
@@ -20,10 +20,10 @@
 import org.apache.kafka.connect.data.Schema;
 
 /**
- * An interface that designates functions that can be indexed using the
+ * Something that can be indexed as if it were a function
  * {@link UdfIndex}.
  */
-public interface IndexedFunction {
+public interface IndexableAsFunction {
```

Comment:
maybe just `FunctionSignature`? I know this was my original naming, but the interface name should probably describe what it is rather than how it's used.

Reply:
done